release-notes.md

```diff
@@ -6,4 +6,71 @@ id: release-notes
 
 # {{ page.title }}
 
+## 3.0.0 *November @, 2016*
+
+It has been a long road but in the end we made it and riot@3.0.0 is finally out and it represents a
+step forward compared to riot 2 for stability, performance and flexibility.
+
+List of improvements and breaking changes
+
+__riot__:
+
+  - _Change:_ build riot using rollup+babel instead of smash using only es6 modules syntax
+  - _Change:_ clean up the tests splitting them into several files, now our unit test was rewritten using chai.js and es6 syntax
```

Comment:
clean up the tests splitting them into several files, now our unit test 'are' rewritten using chai.js and es6 syntax